show/fabric.py

```diff
@@ -74,3 +74,19 @@ def queue(namespace):
     if namespace is not None:
         cmd += ['-n', str(namespace)]
     clicommon.run_command(cmd)
+
+
+@counters.group(invoke_without_command=True)
+def rate():
+    """Show fabric countes rate"""
+    pass
+
+
+@rate.command()
+@multi_asic_util.multi_asic_click_option_namespace
+def mbps(namespace):
```

Comment:
why do we need this? can we have the command `show fabric counters rate` and we can have the default speed as `mbps` ?

Reply:
> why do we need this? can we have the command `show fabric counters rate` and we can have the default speed as `mbps` ?

Thank you for your comments . I just followed one that we discussed in the HLD ( 2.8.3)  ,  let me know if this is ok or want me change to the new command format that you mentioned and update the HLD as well. 

thank you

Reply:
> > why do we need this? can we have the command `show fabric counters rate` and we can have the default speed as `mbps` ?
> 
> Thank you for your comments . I was just following the one that we discussed in the HLD ( 2.8.3) , let me know if this is ok or want me change to the new command format that you mentioned and update the HLD as well.
> 
> thank you

revised after last Friday's discussion, and update the HLD as well.